bip-0111.mediawiki

```diff
@@ -0,0 +1,92 @@
+BIP: 111
+Title: NODE_BLOOM service bit
+Author: Matt Corallo <bip@bluematt.me>, Peter Todd <pete@petertodd.org>
+Type: Standards Track (draft)
+Created: 20-08-2015
+
+Abstract
+========
+
+This BIP extends BIP 37, Connection Bloom filtering, by defining a
+service bit to allow peers to advertise that they support bloom filters
+explicitly. It also bumps the protocol version to allow peers to
+identify old nodes which allow bloom filtering of the connection despite
+lacking the new service bit.
+
+
+Motivation
+==========
+
+BIP 37 did not specify a service bit for the bloom filter service, thus
+implicitly assuming that all nodes that serve peers data support it.
+However, the connection filtering algorithm proposed in BIP 37, and
+implemented in several clients today, has been shown to provide little
+to no privacy[1], as well as being a large DoS risk on some nodes[2].
+Thus, allowing node operators to disable connection bloom filtering is a
+much-needed feature.
```

Comment:
Meh, the reason others didnt implement it is because of the DoS issues, mostly, not because they just didnt feel like it, so thats covered.